codepropertygraph/src/main/scala/io/shiftleft/codepropertygraph/cpgloading/CpgLoaderConfig.scala

```diff
@@ -2,25 +2,27 @@ package io.shiftleft.codepropertygraph.cpgloading
 
 object CpgLoaderConfig {
 
+  // To add a new option, please add a default value below and corresponding
+  // methods in the case class.
+
   def default =
     CpgLoaderConfig(
       createIndices = true,
       onDiskOverflowConfig = Some(OnDiskOverflowConfig()),
-      ignoredProtoEntries = None
     )
 
+  @deprecated("Use CpgLoaderConfig.default.withStorage instead", "Jul 19")
```

Comment:
Adding the deprecation date for the `since` field is a rather odd way, especially since `July 19` is rather ambiguous. Either we don't use that field completely (also in other places) or we should be using a version.

Reply:
I've removed "July 19". However, if I don't at least pass an empty string, I get a compiler warning.